app/scss/components/_nav.scss

```diff
@@ -34,6 +34,15 @@
       transition: opacity .2s ease-in-out .2s;
     }
   }
+
+  @include breakpoint(medium-screen) {
```

Comment:
Since we're refactoring to use the `@include above(` breakpoints (https://github.com/siakaramalegos/clioandcalliope/blob/master/app/scss/_base.scss#L13), can you update these? In my next PR I plan to remove the `@include breakpoint(` lines from `_base.scss`. Happy to lend a hand here if you need :+1:

I think we've also established the convention of naming the breakpoint within the selector, so it'd be nice if you could switch this to:
```
.logo {
  ...
  ...
  @include above(...)
  ...
  }
}
```